src/deploy/scripts/signal_install.sh

```diff
@@ -0,0 +1,17 @@
+git clone -q https://github.com/jeremysquires/signal.git
+cd signal
+git checkout deploy
+
+cd src/api
+npm install
+sudo npm install -g jest
+sudo npm install -g dredd
+
+cd ../menuguide
+npm install
+sudo npm install -g @vue/cli
+sudo npm install -g @vue/cli-service-global
```

Comment:
Initially tried to run the dev server, but it doesn't easily install as a service, so moved to the serve package.